.gitignore

```diff
@@ -2,6 +2,7 @@
 
 # dependencies
 /node_modules
+yarn.lock
```

Comment:
Good catch. The only problem is that this `.gitignore` file is being copied by spraygun into the generated project. We don't want `yarn.lock` to be ignored in the actual generated project.

The robust solution would be to add some logic to `.spraygun.js` to modify the `.gitignore` after project has been generated to remove the `yarn.lock` line? It might be a matter of adding to this list of replacements:

https://github.com/carbonfive/spraygun-react/blob/master/.spraygun.js#L13-L32

Like:

```js
[/^yarn.lock$/, "", ".gitignore"]
```

Reply:
I went ahead and added this to the list, but I don't really know how to test or QA it. Should we add a test asserting that the generated gitignore doesn't contain yarn.lock?

Reply:
I tested by running spraygun with `-p` to use the local working copy of spraygun-react and the generated project had a gitignore without the yarn.lock, as expected. It works!

```
$ npx spraygun -p ../carbonfive/spraygun-react spnoyarn
```